string name
fed_lr_ros/Weights weights
---
